### PR TITLE
feat: Saved carts list page UI #9194

### PR DESCRIPTION
**Pull request by bgambocjaviniar**
closes GH-9194

* contains changes from #11061 as I am branching off that to continue working
* changes related to the branch above will be removed from the commit and rebased once that ticket gets merged to the epic branch

* currently only holds UI changes. The core logic will be done shortly as the team decided to work from UI first then core logic

* contains mocks until core logic is complete

### CODE REVIEW

File: feature-libs/cart/saved-cart/occ/saved-cart-occ.module.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { CostCenterOccModule, provideDefaultConfig } from '@spartacus/core';
+import { defaultOccSavedCartConfig } from './config/default-occ-saved-cart-config';
+
+@NgModule({
+  imports: [CommonModule, CostCenterOccModule],
```
Comment (dydome) [inline feature-libs/cart/saved-cart/occ/saved-cart-occ.module.ts]: CostCenterOccModule should be removed

Comment (bgambocjaviniar) [reply feature-libs/cart/saved-cart/occ/saved-cart-occ.module.ts]: that's from the base feature-lib
will remove it there

File: feature-libs/cart/public_api.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/*
+ * Public API Surface of cart
+ */
+
+export * from '@spartacus/cart/saved-cart';
```
Comment (dunqan) [inline feature-libs/cart/public_api.ts]: We are moving away from the idea of reexporting everything in the top entry point. I would suggest:
```
export {};
```

File: package.json [hunk 3]
```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "build": "env-cmd -e ci,b2c,$SPA_ENV ng build storefrontapp --prod",
     "build:assets": "yarn --cwd ./projects/assets build",
+    "build:cart": "ng build cart --prod",
```
Comment (dunqan) [inline package.json:12]: Why we are not building schematics with the lib?
Also, we probably should include it in build:libs command (the same with 'test:libs).

**Comment by bgambocjaviniar** [comment]
closing this PR for GH-11213

as it got messy from branching

**Pull request closed without merging**